Translate Succubus Carol (10072) (#487)
Giratena/Succubus-Duel-English-Translation: CSV/Language/English/Card/10071.txt

default
生意気キャロル,Cheeky Succubus - Carol
EU71_paizuriAnime,
EU71_bustShakeAnime,

> 生意気キャロルがサキュバス化した姿。もともと大きかった胸はさらに二回りほど大きくなり、その圧倒的な質量は暴力的なほどの快楽を生み出す。胸をわざとらしく揺らすだけで相手をすぐに前かがみにさせることができ、動きが鈍ったところを胸で捕まえてしまえば、顔や性器を挟まれ男はなすすべもなくエナジーを漏らすことになるだろう。,Carol's form after her succubus transformation. Her already large breasts have grown about two sizes bigger; their overwhelming mass now creating violently intense pleasure when used to attack. Just by bouncing her chest; she can make opponents instantly double over; and if she captures them with her breasts while their movements are slowed; any man will have their faces or genitals squeezed between them as their energy helplessly leaks out.

Author: Hmm I like how you've avoided 'you' in this translation! Still, I feel that this description doesn't flow well in English at all. Try something like this: Notice how the same meaning is achieved, but in a way that flows better in English. A good example of how direct translations can fall a little flat. I think I really like how this suggestion turned out ^^

Note: 鈍った is more like 'hindered' rather than 'slowed'

```suggestion
生意気キャロルがサキュバス化した姿。もともと大きかった胸はさらに二回りほど大きくなり、その圧倒的な質量は暴力的なほどの快楽を生み出す。胸をわざとらしく揺らすだけで相手をすぐに前かがみにさせることができ、動きが鈍ったところを胸で捕まえてしまえば、顔や性器を挟まれ男はなすすべもなくエナジーを漏らすことになるだろう。,Carol's new form after being transformed into a succubus. Her already large breasts have grown by about 2 cup sizes; and their overwhelming mass can inflict irresistible pleasure when she attacks. Opponent's keel over at the mere sight of her bouncing breasts; and if she uses her them to capture her enemies while their movements are hindered; those men will have their faces or genitals squeezed between them as their energy helplessly leaks out.
```

[1].１ターンに１度発動できる。相手フィールド上のすべての「おっぱいに弱い」状態のモンスターに「見惚れ」（この状態のモンスターはプレイヤーへの攻撃を防げない）状態を付与する。すでに「見惚れ」状態の場合、「魅了」状態を付与する。,[1]. Once per turn; apply [Mesmerized] (monsters in this state cannot prevent attacks against the player) to all monsters with [Weak to Breasts] status on the opponent's field. If already [Mesmerized]; applies [Charmed] status instead.
[2].1ターンに１度、相手がモンスターを場に出したとき発動できる。そのモンスターに「おっぱいに弱い」を付与する。,[2]. Once per turn when the opponent summons a monster to the field; apply [Weak to Breasts] to that monster.
(戦闘時に弱点を付与する。このカードは「おっぱいに弱い」相手から受ける戦闘ダメージが半分になる。),(Applies weakness during battle. This card takes half battle damage from opponents with [Weak to Breasts].)
あはん❤おっぱいマゾども～❤見てぇ❤\n超爆乳で、前かがみ❤\n深すぎる谷間見せつけながら～、ゆさゆさゆさぁ❤,Ahahn!❤ Hey; you breast-obsessed perverts~❤ Look at these!❤ My massive tits almost spill out of my top when I lean forward.❤ I'll go ahead and show off my impossibly deep cleavage. I'll even bounce them for you! Bouncy bouncy bounce~❤

> 両手でむにゅうっておっぱい持ち上げてぇ❤\nほらぁ、おっぱいマゾども、目そらしてみろよぉ❤\n今からこのおっぱい、どぷんって落としちゃうよ～❤,I'll lift my massive breasts; pushing them up with a squish~❤ Come on; you tit-masochists; just try looking away!❤ I'm about to drop these bad boys so hard~❤

Author: Nice localization at the end!

> あは、おっぱいばっかり見てるのバレバレなんですけどー❤\nエアパイズリで脳みそバカにしたげるよ、うりうり～❤,Aha; I can tell you're staring at my tits. You're so obvious~❤ I'll give the air a titjob and make you all dummies! Rub rub~❤

Author: "I'll give the air a titjob" might be a bit flat?

Author: ```suggestion
あは、おっぱいばっかり見てるのバレバレなんですけどー❤\nエアパイズリで脳みそバカにしたげるよ、うりうり～❤,Aha; I can tell you're staring at my tits. You're so obvious~❤ Time to bounce my tits up and down and turn you all into dummies! Rub rub~❤
```
Maybe something like this?

Reviewer: Hmmm I like the direction. How about 'I'll bounce and squish my 'till you ask turn into dummies'

Author: 'I'll bounce and squish my tits 'till you ask turn into dummies' sounds good ^^

ぷぷっ、乳揺らしただけで見とれてメロメロ～❤\nおっぱいひとつで何にもできなくされて、\n恥ずかしくないの？チョロすぎじゃん❤きゃはは❤,Fufu; just bouncing my tits made you stare and fall totally head-over-heels~❤ Left completely helpless from just one pair of breasts... Aren't you embarrassed? You're way too easy!❤ Kyahaha~❤
くすくす❤どんだけおっぱい好きなんだっての❤\n目をハートにして棒立ちじゃん❤おっぱいひとつで何にもできなくされて、\nよっわ～い❤きゃはは❤,*Giggle*❤ Just how much do you love tits anyway?❤ Standing there frozen; staring at my chest all lovey-dovey.❤ Left completely helpless from just one pair of breasts... So weak!❤ Kyahaha~❤
やっほー。新しいモンスターくん❤\n歓迎にほら、わたしの谷間いっぱい見ていいよぉ～❤,Yoohoo! Hey; new monster boy.❤ As a way of saying welcome; look; I'll let you stare at my cleavage all you want~❤
モンスターくんがじーって見てる間、\nゆさゆさって揺らしたげる❤\n何秒ぐらい見続けちゃうかなぁ❤,And now that your eyes are; like; totally glued to them; I'll even give them a sway; just for you~❤ How long do you think you'll keep staring?❤
新しいモンスターくんに、おっぱいサービス❤\n肘でぎゅって谷間押し上げてぇ、\nほら、見てみてぇ～❤,Titty service time for the new monster boy!❤ I'll push my cleavage up with my elbows; so go on; take a look~❤

> 召喚されたばかりのところ悪いけどぉ、\nおっぱいに弱くしちゃうね～❤\nほら、エアパイズリでゆさゆさ～❤,Sorry to do this right after you 've been summoned; but I'm going to make you totally weak to breasts; okay?❤ Watch; I'll give the air a titjob! Bouncy bouncy~❤

Author: "I'm going to make you totally weak to breasts" sounds a bit weird but it's not something I'd block over.

Maybe something like this would flow better?:
```suggestion
召喚されたばかりのところ悪いけどぉ、\nおっぱいに弱くしちゃうね～❤\nほら、エアパイズリでゆさゆさ～❤,Sorry to do this right after you 've been summoned; but I'm going to turn you into a hopeless boobie addict now; okay?❤ Watch; I'll rub my boobs together; kinda like they were wrapped around something big and hard; and... Bouncy bouncy~❤
```

あはぁ、めっちゃガン見じゃん❤\nもっともっと見てぇ、おっぱい大好きなマゾになっちゃえ❤\nきゃはは❤,Ahaha; you're totally staring!❤ That's it; keep staring and become a breast-loving pervert.❤ Kyahaha~❤

> くす❤鼻息あら～い❤\n谷間見てここにチンポ挟まれた～いとか想像したの？\nやってほしいなら、ほらぁ、チンポ入れちゃっていいよ？きゃはは❤,*Giggle*❤ Your breathing's getting so heavy~❤ Were you imagining your cock getting trapped in my cleavage? If you want it that bad; come on; why not go ahead and put your cock in?❤ Kyahaha~❤

Author: ```suggestion
くす❤鼻息あら～い❤\n谷間見てここにチンポ挟まれた～いとか想像したの？\nやってほしいなら、ほらぁ、チンポ入れちゃっていいよ？きゃはは❤,*Giggle*❤ Your breathing's getting so heavy~❤ Were you imagining your cock getting trapped inside my cleavage? If you want it that bad; come on; why not go ahead and slip your cock in?❤ Kyahaha~❤
```

ねえ、ねえ～❤戦いなんてやめてさぁ、\nわたしたちに降参しちゃおうよ～❤\nこのおっぱい見たら分かるでしょ？男じゃ勝つのムリだって❤,Hey; hey~❤ You should totally stop fighting and just; like; surrender to us instead!❤ Looking at these tits. You get it; don't you? There's no way a man could win~❤
だってさー、頭もチンポも余裕で全部挟めちゃうんだよ？\n一回挟まれたら、身をよじってもジタバタしても、\nデカすぎおっぱいの中からは逃げられない❤絶対射精させられちゃう❤,I mean; I could easily crush both your head and cock between these things; you know? Once you're trapped; no matter how much you writhe or struggle; there's no way you could escape a chest this oversized.❤ I'd definitely force you to cum~❤
くすくす❤今の話聞いて、チンポビンビンにしちゃってるし❤\nほら、爆乳おっぱいでなでなでしたげるから\nさっさと降参してエナジーよこしなよ❤マゾチンポ❤きゃはは❤,*Giggle*❤ Just hearing about it got your cock rock hard.❤ Here; I'll go ahead stroke your cock with my bombshell tits; so hurry up and surrender your energy to me.❤ Dirty pervert cock!❤ Kyahaha~❤
顔をおっぱいに埋めて～❤\nむにゅむにゅ～❤ぱふぱふ～❤\nあはぁ、一瞬でふにゃふにゃじゃん❤あたしの爆乳で蕩けちゃった？,I'll bury your face in my chest.❤ Squish squish~❤ Puff-puff~❤ Ahaha; you went limp right away!❤ Did my massive tits make you melt?

> 力全然入んないんでしょ？\nおっぱい好きだもんねー❤こんな爆乳押し付けられちゃったら、\nおっぱいで頭いっぱい❤攻撃とか防御とか考えられないよねー❤,What's wrong? Can't put any strength into your body? You really are a boob-lover; after all~❤ The instant huge tits like these get pressed against you; your head's stuffed with nothing but breasts.❤ You can't think about fighting back at all; can you?❤

Author: "your head's stuffed with nothing but breasts" seems like a forced leap of logic from the previous sentence part.

```suggestion
力全然入んないんでしょ？\nおっぱい好きだもんねー❤こんな爆乳押し付けられちゃったら、\nおっぱいで頭いっぱい❤攻撃とか防御とか考えられないよねー❤,What's wrong? Can't put any strength into your body? You really are a boob-lover; after all~❤ The instant my huge breasts are pressed against you; your head becomes flooded with images of my tits.❤ You can't think about fighting back at all; can you?❤
```

Reviewer: Hmm... 'Images of my tits' feels really weird in context here. Maybe something more like 'your silly little head drifts off fantasizing about my tits.' or something?

Author: Hmm I feel like that's too wordy to get the point across. I've seen the phrase "flooded with images of" quite commonly in writing like as in MGD, so I think it's appopriate to use ^^

Reviewer: The catty side of me wants to day revalence doesnt equal quality, but in communities like this normalization and precedence carry weight.

Author: See the problem is when I hear the phrase "your head's stuffed with nothing but breasts" it causes the reader to think about that literally, especially since it's not a real phrase. "flooded with images of" is the writing standard here.

Author: Went with: You're totally helpless now; aren't you?❤ You love boobs so much; when they're pressed against you like this; your thoughts become filled with my tits~❤ No room to think about attacking or defending at all~❤

プライドも仲間もすてておっぱいに降伏しちゃって恥ずかしくないの？\nぷっ、そんなことよりチンポ気持ちよくなるほうが大事か❤\nザコってホント終わってるね❤,Abandoning your pride and companions to surrender to tits... Aren't you embarrassed? *Snort* I guess your cock feeling good is more important than any of that.❤ Weaklings like you really are hopeless; aren't they?❤
ほら、エナジー、さっさとよこしな？\nこの胸をたっぷり使って、速攻で搾り取ってやるよ❤,Come on then. Hurry up and give me your energy! I'll use these tits to squeeze you dry in no time~❤
はい、瞬殺❤弱っちいチンポ、\nアタシのおっぱいに全然耐えられなかったね❤くすくす❤,There; instant victory!❤ Your pathetic cock couldn't withstand my breasts at all; huh?❤ *Giggle*❤
サキュバスになってデカくなったこの爆乳、\n最高に気持ちよかったでしょ？\nうわぁ、すっごくみっともない顔で死んでる❤,These massive tits felt even better now that becoming a succubus made them grow so much; huh? Oh man; you died with a totally pathetic expression on your face!❤
くす、脚に何か熱いのを感じるんだけど❤\nぱふぱふってされただけでこんなに射精しちゃったんだ。\nザーコ❤,*Giggle* I feel something hot dripping down my leg.❤ Did you seriously cum this much from just a Puff-Puff? Weakling~❤
あううっ…❤我慢できない…❤,Ahhh...❤ I-I can't resist...❤
きゃは❤なになに～❤\nエロいポーズでおっぱい見せられて、\n我慢できなくなったわけ？,Kyaha~❤ What's the matter?❤ Couldn't hold back after seeing my tits in a sexy pose?

> ちょっとおっぱい揺らして誘惑されただけなのに、ちょっろ❤\n男ってチンポでものを考えてるから、ほんと弱っちいよね❤,Just a little breast bouncing and a bit of seduction; and you're totally hooked~❤ Men just think with their cocks; so they're all really pathetic; huh?❤

Author: ```suggestion
ちょっとおっぱい揺らして誘惑されただけなのに、ちょっろ❤\n男ってチンポでものを考えてるから、ほんと弱っちいよね❤,Just a little breast bouncing and a bit of seduction; and you're totally hooked~❤ Men only think with their cocks; so they're all really pathetic; huh?❤
```

きゃは❤なになに～❤\nわたしのデカすぎるおっぱい見てたら、\n我慢できなくなったわけ？,Kyaha~❤ What's the matter?❤ Couldn't hold back after staring at my oversized tits?

> そんなに手を早く動かしてぐちゅぐちゅ扱いちゃってさー❤\nおっぱいをおかずにシコシコ❤今がどういう状況か分かってる？\nアンタたちサキュバスに食べられようとしてるんだよ？,You're jerking yourself so fast it's making these weird; wet noises~❤ Staring at my breasts while stroking your cock like that...❤ Do you even realize the situation you're in? You know you're about to be devoured by a succubus; right?

Author: ```suggestion
そんなに手を早く動かしてぐちゅぐちゅ扱いちゃってさー❤\nおっぱいをおかずにシコシコ❤今がどういう状況か分かってる？\nアンタたちサキュバスに食べられようとしてるんだよ？,You're jerking yourself so fast I can hear weird; wet noises~❤ Staring at my breasts while stroking your cock like that...❤ Do you even realize the situation you're in? You know you're about to be devoured by a succubus; you know?
```

Reviewer: I don't know that I like 'i can hear' as a substitute.

Author: Hm I think the issue here is that it's not obvious what "it" is that's making the noises.

Reviewer: So how about 'jerking your cock' and then 'it's making these weird noises'

Author: Yep that works ^^

> それなのにそんなに興奮するとか、バッカじゃないの？\nきゃはは❤脳みそ空っぽすぎでしょ❤,And yet you're still getting this excited. Are you stupid?❤ Kyahaha~❤ Your brain is way too empty; dontcha think?❤

Author: Way too empty is a weird phase to use

Reviewer: What about something along the lines of just 'your heads totally empty; huh?'

Author: 'your **heads'** totally empty; huh?' could work yeah!

あはん、ほらもっとおっぱいの匂い嗅いでいいよ❤\n口からよだれを垂らしてるそのアホ面、\nもっともっと間抜けな顔にしなよ❤バーカ❤,Ahahn; that's it. Feel free to get an even better whiff of my tits~❤ I'll make that stupid drooling face of yours even dumber!❤ Idiot~❤
あうう、あううっ❤気持ちいいっ、ああっ❤❤,Aghh; ahhhh!❤ It feels so good! Ahhh!!❤❤
くす❤間抜けだバカだって言われているのに、うれしそうな顔❤\nサキュバスフェロモンで頭おかしくなってるのかな～。\nそれとも元からマゾなのかなー❤きもーい❤,*Giggle*❤ I'm calling you things like "idiot" and "dumb"; but you still look so happy~❤ Are you going crazy from my succubus pheromones? Or were you always a masochist?❤ Gross~❤
そんなに興奮してシコシコしてたら、\nあっという間にアンタの命が終わっちゃうよ？\nこのエローいおっぱいに向けて、どびゅって命を出して死んじゃうよ？,If you stay this excited and keep stroking yourself like that; you'll die. You know that; right? You'll shoot out your very life essence toward these sexy tits and totally drop dead!
あはは❤死ぬかもって思っても、しこしこやめられないねー❤\nこのおっぱいに精液かけようと、必死になってしこしこしこしこ❤,Ahaha~❤ Even knowing that; you can't stop stroking; can you?❤ You desperately want to cover my chest with your cum; so you just stroke stroke stroke~❤
ほらバーカ❤\n思い切りおっぱい揺らしてあげるから、\nさっさとイってエナジー全部おっぱいに食べられろ❤,Then do it; idiot!❤ I'll bounce my tits as hard as I can; so hurry up and cum; and let them devour all your energy~❤

> この谷間にぜんぶ投げ出しちゃえ❤マーゾ❤,Give everything you have to my cleavage~❤ You masochist!❤

Author: ```suggestion
この谷間にぜんぶ投げ出しちゃえ❤マーゾ❤,Shoot everything you have out for my cleavage~❤ You masochist!❤
```

Reviewer: How about we break it up for effect? "Shoot out everything you have! Surrender it all to my cleavage/tits~'

Author: Yeah that could work ^^

はひぃっ・・・❤,Hahhhh...❤

> きゃは、見てよこの量❤\nわたしのおっぱいがどろどろ～❤\nほら、これをおっぱいで挟んでぐちゅぐちゅって潰してあげる❤,Kyaha; look at this load!❤ You got my tits all sticky~❤ Watch; I'll squeeze you jizz between my breasts and crush it all up~❤

Author: ```suggestion
きゃは、見てよこの量❤\nわたしのおっぱいがどろどろ～❤\nほら、これをおっぱいで挟んでぐちゅぐちゅって潰してあげる❤,Kyaha; look at this load!❤ You got my tits all sticky~❤ Watch; I'll squeeze your jizz between my breasts and crush it all up~❤
```

> って、おっぱいで命をもぐもぐされて、興奮してるの？\nチンポビクビクさせちゃっててきもーい❤\n死にかけなのにこんなの見て興奮するとか、頭悪すぎでしょ❤,Wait; are you getting excited watching your life essence get devoured by my tits? Your cock is twitching! Totally gross~❤ Even on the verge of death you're getting aroused just watching this. You're way too stupid!❤

Author: ```suggestion
って、おっぱいで命をもぐもぐされて、興奮してるの？\nチンポビクビクさせちゃっててきもーい❤\n死にかけなのにこんなの見て興奮するとか、頭悪すぎでしょ❤,Wait; are you getting excited watching your life essence get devoured by my tits? Your cock is twitching! Totally gross~❤ Even on the verge of death you're getting aroused just watching this. What a loser!❤
```

ううっ…❤我慢できない…❤,Ugh...❤ I can't hold back...❤

> あはは、アンタもオナニーしちゃうんだ❤\nわたしのおっぱい凶悪すぎじゃない？\nどんどん男の子が勝手に負けていってウケるんだけど❤,Ahaha; you're totally masturbating; too~❤ Are my tits really that huge? Boys keep losing on their own to them. It's totally hilarious!❤

Author: ```suggestion
あはは、アンタもオナニーしちゃうんだ❤\nわたしのおっぱい凶悪すぎじゃない？\nどんどん男の子が勝手に負けていってウケるんだけど❤,Ahaha; you're totally masturbating; too~❤ Are my tits really that huge? Boys keep losing on their own to them. It's hilarious~❤
```

分かってるのかなー❤闘技場のお遊びバトルとは違うんだよ？\n負けたらそのまま吸い殺されちゃう、こわーいサキュバスとのバトル❤\nそれなのに、チンポをシコシコ❤シコシコ❤,You get where you're at; right?❤ This isn't some playful little arena match. This is a battle with a terrifying succubus; ready to drain you to death if you lose~❤ And you're still just stroking your cock!❤ Stroke stroke~❤

> あー、そっかおっぱい見てたら頭バカになって、\n負けたらどうなるかとか、もう考えられないんだ❤\nおっぱいのことで頭がいっぱいなんでしょ？ほら・・・❤,Ah; I get it. Watching my chest made your brain go stupid; so now you can't even think about what happens if you lose anymore~❤ Your mind is totally filled with thoughts of tits; isn't it? Then here...❤

Author: ```suggestion
あー、そっかおっぱい見てたら頭バカになって、\n負けたらどうなるかとか、もう考えられないんだ❤\nおっぱいのことで頭がいっぱいなんでしょ？ほら・・・❤,Ah; I get it. Watching my chest made your brain go stupid; so now you can't even think about what happens if you lose anymore~❤ Your mind is totally filled with thoughts of my tits; isn't it? Then here...❤
```

むぎゅうっておっぱい寄せて、アンタの顔に近づけてー・・・❤,I'll press my breasts together tight and bring them close to your face...❤
おっぱいゆさゆさゆさ～❤あはーん❤\nサキュバスになって大きくなったアタシのおっぱい❤\nたっぷり見て悩殺されろ❤ザーコ❤,And make them bounce bounce bounce!❤ Ahan~❤ They grew so big when I became a succubus.❤ Stare all you want and be charmed to death~❤ Weakling!❤

> あうううっ、すごく揺れてぇ・・・❤\nあうっ、ああっ、いい匂いでっ❤ああっ❤❤,Agh; they're bouncing so much...❤ Ah; ahh; they smell so good...❤ Ahhh!❤❤

Author: ```suggestion
あうううっ、すごく揺れてぇ・・・❤\nあうっ、ああっ、いい匂いでっ❤ああっ❤❤,Agh; they're bouncing so much...❤ Ah; ahh; and they smell so good...❤ Ahhh!❤❤
```

きゃはは❤おちんちん扱くのはやぁい❤\n男ってほんとバカばっかりだよね❤脂肪のかたまりがそんなに大事？\n死んでもいいからおっぱいに負けたいとか、アホすぎでしょ❤,Kyahaha~❤ You're stroking your cock so fast!❤ Men really are all idiots; aren't they?❤ Are these sexy lumps of fat really that important? Wanting to lose to tits; even if it means death... You're way too stupid~❤
自分の命とかよりもおちんちんシコシコする方が大事❤\nエロいおっぱい揺らされたら、それで脳みそ空っぽ❤\n知能がゼロになって死ぬまでシコっちゃう❤,Stroking your cock is more important to you than your own life now.❤ Once these sexy breasts start bouncing; your brain goes totally blank~❤ Your intelligence drops all the way to zero and you'll just stroke yourself to death!❤

> 今だっておっぱいが揺れるのに合わせて、しこしこ、しこしこ、\n口からよだれを垂らしながら扱いてる❤\nきもすぎなんだけどその顔❤きゃははは❤,Even now; you're stroking in rhythm with my bouncing tits. Stroke stroke~❤ Oh my god; you're even drooling a little!❤ That face is way too gross!❤ Kyahahaha~❤

Author: ```suggestion
今だっておっぱいが揺れるのに合わせて、しこしこ、しこしこ、\n口からよだれを垂らしながら扱いてる❤\nきもすぎなんだけどその顔❤きゃははは❤,Even now; you're stroking in time with my bouncing tits. Stroke stroke~❤ Oh my god; you're even drooling a little!❤ That face is way too gross!❤ Kyahahaha~❤
```

理性を振り絞って攻撃した,Summon your will and attack with full force
深い谷間から目が離せなかった,'So big... and deep...'
ぷっ❤完全にクギ付け❤\n1ミリも目をそらせないではぁはぁしちゃってさ❤\nバカ面すぎでしょ❤,Pfft~❤ You're totally transfixed!❤ You can't look away even one little bit; and you're actually panting!❤ What a stupid face~❤
それじゃさっそく、アンタの脳みそおかしくしてあげる❤\nわたしの声を聞きながら、おっぱいガン見しな❤\nおっぱい催眠かけてやるよ❤,Then let's get started. I'll make your brain go crazy!❤ Listen to my voice while staring at my breasts.❤ I'll hypnotize you with my tits~❤
洗脳の乳揺れ,Brainwashing Breast Bounce
おっぱいの揺れを見てると、頭ボーっとしてくるよね❤\nそれはアンタの頭がおっぱいに埋められていってる証拠❤\n考える力をおっぱいに奪われていっているの❤,Watching my breasts bounce makes your head feel fuzzy; right?❤ That means your head is being filled by my breasts!❤ Your ability to think is being stolen by my tits~❤
おっぱいが一回揺れるたびに、アンタの脳はどんどんバカになる❤\n理性だけじゃなくて思考能力も落ちていって、\n簡単な計算もできなくなる❤,Every time my breasts bounce; your brain gets stupider and stupider.❤ Not just your willpower; but your ability to think drops too; until you can't even do simple math anymore~❤

> 頭が悪くなると、何も自分で判断できなくなっちゃうんだよ❤\n自分で考えられないから、人の言いなりになっちゃうの❤,When that happens; you can't make any decisions on your own anymore!❤ Since you can't think for yourself; you become obedient to others~❤

Author: よ is used when conveying new or otherwise unknown information. It doesn't always translate to "!"
```suggestion
頭が悪くなると、何も自分で判断できなくなっちゃうんだよ❤\n自分で考えられないから、人の言いなりになっちゃうの❤,And when you get into a state like that; making decisions becomes impossible~❤ All nice and obedient because you can't think for yourself anymore.❤
```

Reviewer: Some of the wording sounds weird for Carol. How about:

And when you get like that; making decisions becomes impossible.❤ You end up nice and obedient; all because you can't think for yourself anymore~❤

Author: Yep that also works ^^

> アンタの頭の中を埋めているおっぱいに、思考を支配されちゃう❤\n言いなりにならないようにしないといけないけど、\n頭空っぽだから抵抗できない❤,The breasts filling your head will dominate your thoughts.❤ You need to avoid becoming obedient; but your head is just so empty you can't resist~❤

Author: A bit forced and stiff here. Head is repeated twice.
```suggestion
アンタの頭の中を埋めているおっぱいに、思考を支配されちゃう❤\n言いなりにならないようにしないといけないけど、\n頭空っぽだから抵抗できない❤,My breasts are already filling your head and dominating your thoughts~❤ You need to avoid becoming obedient; but your mind is just so empty right now you can't even bring yourself to resist~❤
```

あう、あうう・・・❤おっぱい・・・❤ああっ・・・❤,Ah; ahhh...❤ Breasts...❤ Ahhh...❤
あは、いい感じに催眠がかかってきた❤\n目がとろーんってとろけて、口もあんぐり開いて、\nぷぷっ、メロメロなのが分かりやすすぎ❤,Aha; the hypnosis is working perfectly!❤ Your eyes are all droopy; and your mouth is totally hanging open. Pfft; you're so obviously head-over-heels for me~❤

> おっぱいを揺らされると気持ちいいよね❤\nおっぱいを揺らされるだけで何も考えられない❤\n支配されるのが気持ちよくて、なんでも言うことを聞きたくなる❤,Watching my breasts bounce feels good; doesn't it?❤ Just seeing them move like this makes you unable to think of anything.❤ Being dominated feels so good you'll do whatever I say~❤

Author: It's a bit weird, but it flows better to describe it like this. It's not the watching that feels good, but rather the act of watching that feels good.
```suggestion
おっぱいを揺らされると気持ちいいよね❤\nおっぱいを揺らされるだけで何も考えられない❤\n支配されるのが気持ちよくて、なんでも言うことを聞きたくなる❤,It feels good to watch my breasts bounce; doesn't it?❤ Just seeing them move like this makes you unable to think of anything.❤ Being dominated feels so good you'll do whatever I say~❤
```

> ほら、とどめだよ❤\nわたしが傷ついた分をドレインしながら、\nおっぱいで脳みそぶっ壊してあげる❤,And now; for the finishing blow!❤ I'll drain you for all the damage I took and totally destroy your brain with my tits!❤

Author: ```suggestion
ほら、とどめだよ❤\nわたしが傷ついた分をドレインしながら、\nおっぱいで脳みそぶっ壊してあげる❤,And now; for the finishing blow!❤ I'll drain you for all the damage I took as I use my tits to totally break your mind~❤
```

理性が飛んで身動きできなかった,'So big... I... I can't move...'
げふっ、嘘でしょ…。まだそんなスピードが出せるなんて…。,Gah; no way... You can still move that fast...
きゃうっ！ううっ、理性強すぎ…。,Kyah! Ugh; your will is way too strong...
はい、捕まえた❤\n勃起チンポが邪魔すぎて動けなかった？\n違うよね、挟まれたくて足止めちゃったんだよね❤,There; caught you~❤ Were you stuck in place because your hard cock got in the way? No; that's not it; you stopped yourself because you wanted to be milked; didn't you?❤
サキュバスの前で欲望に流された男の末路なんて決まってるの❤\n搾り取られて、干からびて死んじゃうんだよ❤\nほら❤,The fate of men who give in to desire before a succubus is already decided; you know.❤ They get drained dry; withering away into nothing.❤ Now come on~❤
わたしの超デカいおっぱいに挟まれて、\nさっさと死んじゃえ、ザーコ❤,Hurry up and die between my massive tits; you weakling!❤
ぷっ、挟まれたばっかりなのに、もう射精の準備始めちゃってる❤\n焦らしてたからすっごい気持ちいいでしょ？\n気持ちよすぎてチンポ溶けそうなんじゃない？,Pfft; I've just barely squeezed you between them and you're already getting ready to cum.❤ Since I've been teasing you so much; it feels extra nice; huh? Does it feel so good your cock might melt?
我慢なんてさせてあげない❤\nさっさと出してわたしを回復させてよ。ほらほらほら❤,I won't let you hold back!❤ Hurry up and cum so I can restore my energy! Come on~❤
はひぃっ・・・❤❤,Hahhh...❤❤

> はい、楽勝❤\nサキュバスのおっぱいでチンポ瞬殺されちゃったね❤\n快感強すぎて腰抜けちゃった？くすくす❤,There; easy peasy~❤ Your cock got destroyed instantly by my succubus tits.❤ Did it feel so good your legs give out? *Giggle*❤

Author: ```suggestion
はい、楽勝❤\nサキュバスのおっぱいでチンポ瞬殺されちゃったね❤\n快感強すぎて腰抜けちゃった？くすくす❤,There; easy peasy~❤ Your cock broke instantly between my succubus tits.❤ Did it feel so good your legs give out? *Giggle*❤
```

腰が抜けても関係ないよ。どうせもう体動けなくなるし❤\nエナジー吸われてあっという間に干からびて。\n気持ちよく殺してもらえてよかったね❤くすくす❤,Though it doesn't really matter either way. Your body's going to stop working soon anyway.❤ Your energy'll get sucked out and you'll wither away in no time. Hey; at least you got to die feeling good; right?❤ *Giggle*❤
あは、一発出しただけじゃまだ死なないんだ❤\nそれじゃ死ぬまでいじめてあげるね❤\nオナ猿ザコマスター❤きゃはは♪,Aha; but one load isn't enough to kill you.❤ So I'll go ahead and torment you until you drop dead; okay?❤ You stupid little masturbation-fetish master!❤ Kyahaha~♪

> おっぱいマゾってあわれだねー。\n勝てそうな勝負でもおっぱいに見惚れて負けちゃうんだもん❤\nぷっ、くすくす❤,Tit masochists are so pathetic. Even in battles they could win easy; they all get mesmerized by tits and end up losing~❤ Pfft. *Giggle*❤

Author: ```suggestion
おっぱいマゾってあわれだねー。\n勝てそうな勝負でもおっぱいに見惚れて負けちゃうんだもん❤\nぷっ、くすくす❤,Tit masochists are so pathetic. Even in battles they could win easy; they all get mesmerized by a pair of nice tits and end up losing~❤ Pfft. *Giggle*❤
```

ぷぷっ、おっぱい揺らしてるだけで逆転勝利♪\n男ってチョロすぎでしょ❤,Pfft; all I had to do was bounce my tits and I got a total reversal!♪ Men are way too easy~❤
あんっ、すっごい量が谷間に出てる❤\nわたしのパイズリテクどうだった？\nチンポメロメロにされてすぐ負けちゃったね❤きゃはは♪,Ahh; you're shooting so much out into my cleavage.❤ How'd you like my titfuck skills? Your cock fell totally head-over-heels and you lost so fast!❤ Kyahaha~♪
あはは、どぴゅどぴゅエナジー出しちゃった❤\nおっぱいで顔を挟まれて簡単に昇天❤\nサキュバスおっぱいの力、よく分かった？ザーコ❤,Ahaha; you're spurting out energy like crazy!❤ Looks like getting your face squeezed between tits sent you right to heaven~❤ Do you finally understand the power of a succubus's tits? Weakling~❤
ぷっ、楽勝すぎ❤\n憎いサキュバスが相手だっていうのにこれだもん❤,Pfft; way too easy!❤ Even though your opponent was an evil succubus; look what you let happen.❤

> わたしのおっぱいそんなにヤバかった？\nすっごいとろけ顔でびゅくびゅく射精しちゃってさ、\nこのマーゾ❤,Were my tits really that dangerous? They left you spurting cum everywhere with that melted expression on your face! You masochist~❤

Author: that melted expression is a bit weird
```suggestion
わたしのおっぱいそんなにヤバかった？\nすっごいとろけ顔でびゅくびゅく射精しちゃってさ、\nこのマーゾ❤,Were my tits really that dangerous? You're spurting cum everywhere; and you're making such a slutty face too. You masochist~❤
```

Reviewer: Two problems: 1, I'm pretty sure it's past-tense, and your current suggestion makes it sound like he's still cumming, and 2, I just really hate the term slutty face.

Author: 1. Maybe change spurting to spurted
2. I really hate the term "melted expression" Mainly because it's a pretty engrishy phrase. xD

Reviewer: Okay, let's kill both then, what about something like 'and you've gone all slack-jawed, too'

Author: Yep that works!

あううっ・・・❤くそぉ・・・❤,Agh...❤ Damn it...❤
サキュバス化した生意気キャロルが僕をにやにやと見下ろしている。僕はなんとか体に力を入れようとするが、まったく力が入らない。,The bratty Carol—now transformed into a succubus—looked down at me with a smirk. I tried to somehow force my body to move; but I didn't have any strength left.

> 彼女の胸が与えたあまりに強烈な快感がまだ腰に残っている。僕の脚はぷるぷると震え、地面に尻もちをついたまま立ち上がることもできなかった。,The intensely powerful pleasure her breasts had given me still lingered in my hips. My legs trembled and shook; unable to hold my weight; leaving me sitting on the ground.

Author: When describing things from the MC's perspective, using words like "feel" can be more engaging to the reader.
Also, "leaving me sitting" is a bit weird. 
```suggestion
彼女の胸が与えたあまりに強烈な快感がまだ腰に残っている。僕の脚はぷるぷると震え、地面に尻もちをついたまま立ち上がることもできなかった。,I could still feel the intensely powerful pleasure her breasts had given me lingering in my hips. My legs trembled and shook; unable to hold my weight; forcing me to fall to the floor.
```

きゃはは、なにその脚の動き。どんだけだよ❤\nサキュバスになって、わたしヤバいぐらい強くなってるじゃん❤,Kyahaha; what's with those sad; shaky legs? So pathetic~❤ Now that I've become a succubus; I guess I've gotten ridiculously strong.❤
元からおっぱいおっきかったけどさー、\n今じゃ超爆乳になって、おっぱいひとつでアンタの頭より大きいよ。\nヤバくないこのデカさ❤,My tits were already big before; but now they're super massive—each ones bigger than your head! Pretty awesome; right?❤
Bランクで、無敗だった男を瞬殺❤\nアタシのマスターがずっとアンタに追いつけないのを悩んでたけど、\nぷぷっ、こんな簡単な話だったんだね❤,Big enough to kill an undefeated B-rank man; even.❤ My master was always worrying about not being able to catch back up to you; but pfft; looks like it was this easy all along~❤
サキュバスになっちゃえば、男なんてみんなザコ❤\nデカちち揺らしてちょっと遊んでやれば、\n簡単に倒せちゃう❤あーん、チョロすぎ❤,Once you become a succubus; all men turn into weaklings.❤ Just bounce your big tits and tease them a little; and deafting any one of them is a cinch~❤ Ahh; it's so pathetically easy!❤

> {主人公}さん、ダメです！諦めないで！\nそんなサキュバスの快楽になんか負けちゃダメ！,{主人公}; no! Don't give up! You can't lose to the pleasure from that succubus!

Author: ```suggestion
{主人公}さん、ダメです！諦めないで！\nそんなサキュバスの快楽になんか負けちゃダメ！,{主人公}; no! Don't give up! You can't just let pleasure from some succubus beat you!
```

妹さんのためにこれまで頑張ってきたんでしょう！\n憎い仇のサキュバスに、負けてどうするんですか！\n{主人公}さん！,You've worked so hard for your sister's sake up until now! How can you lose to a no-good succubus?! {主人公}!
向こうで受付さんと他のカードマスターたちの声がする。僕に向かって何か励ましの言葉のようなものを投げかけている。,Somewhere nearby; I can hear the voices of the receptionist and the other cardmasters. They're shouting some kind of encouragement; pushing me to get back up.

> しかし、受付さんの声すら、僕には夢うつつのようにしか聞こえていなかった。目の前でゆっくりと波打っているキャロルの胸に視線を吸われ、思考はもやがかったように何も考えられない。,But even the receptionist's voice sounds so distant to me. My gaze is absorbed by Carol's breats slowly swaying in front of me; and my thoughts are lost in a haze. I can't think of anything but them.

Author: ```suggestion
しかし、受付さんの声すら、僕には夢うつつのようにしか聞こえていなかった。目の前でゆっくりと波打っているキャロルの胸に視線を吸われ、思考はもやがかったように何も考えられない。,But the receptionist's voice sounded distant. My gaze was already focused on Carol's breasts slowly swaying in front of me; and I could feel my thoughts fading. I couldn't think about anything else.
```

> ああ・・・、見ているだけなのに、また体が気持ちよくなってくる。快感がじんわりとせりあがってきて全身に回り、脳の奥まで痺れるようになってしまう。,Ahh... just watching them makes my body feel good again. Pleasure gradually wells up and spreads through me; making even the deepest parts of my brain tingle.

Author: Hmm this inner monologue feels quite stiff. Something like this would probably convey the tone a bit better!

```suggestion
ああ・・・、見ているだけなのに、また体が気持ちよくなってくる。快感がじんわりとせりあがってきて全身に回り、脳の奥まで痺れるようになってしまう。,Ahh... even just looking at them was making my body feel all warm and tingly again. Waves of pleasure began to ripple across my body and towards my crotch; making even the deepest parts of my brain tingle.
```

ぷっ、アンタ、仲間の呼びかけも聞かないでおっぱい見てるの？\nあははは、そんなにあたしのおっぱいが忘れられないんだ❤,Pfft; are you seriously staring at my tits instead of listening to your friends' cries for help? Ahahaha; you really are totally obsessed~❤
いいよ、もっかいやってあげる❤\n最低のおっぱいマゾのアンタを、\nあたしのサキュバスおっぱいで気持ちよくしてあげる❤,In that case; I'll go ahead and make you feel good with my succubus breasts all over again.❤ You pathetic little tit-masochist~❤
いいよ、今度は直接挟んであげる❤\n最低のおっぱいマゾのアンタを、\nあたしのサキュバスおっぱいでバカみたいにアヘらせてあげる❤,In that case; how about I use my succubus breasts and milk you for real this time?❤ You pathetic little tit-masochist~❤

> あっちのやつらが見ている前でさぁ、\nカッコ悪ーい喘ぎ声上げながら、\nおっぱいでエナジー搾り取られな？えい❤,Moan pathetically in front of everyone as your energy drained by my tits; okay? Now take this!❤

Author: ```suggestion
あっちのやつらが見ている前でさぁ、\nカッコ悪ーい喘ぎ声上げながら、\nおっぱいでエナジー搾り取られな？えい❤,Moan pathetically in front of everyone as I use my tits to drain your energy; okay? Now take this!❤
```

あひっっ、あっ❤あっ❤ああっ❤,Ahiii; ah...❤ Ah!❤ Ahhh!❤

> キャロルの胸を見て限界まで勃起していた僕のペニスは、巨大な胸に丸ごと飲み込まれてしまった。柔らかいのにぎゅうっと圧力たっぷりに包み込んできて、腰が溶けそうに気持ちがいい。ペニスから訳が分からないほどの快感が流れ込んでくる。,My penis—which had gotten fully erect again just staring at Carol's breasts—was swallowed completely by her massive chest. Despite their plush feel; they wrapped around me with intense pressure; feeling so good my hips might melt. Incomprehensible pleasure flowed from my penis into the rest of my body.

Author: Another case of the dialogue being too stiff. Here's a better suggestion:
```suggestion
キャロルの胸を見て限界まで勃起していた僕のペニスは、巨大な胸に丸ごと飲み込まれてしまった。柔らかいのにぎゅうっと圧力たっぷりに包み込んできて、腰が溶けそうに気持ちがいい。ペニスから訳が分からないほどの快感が流れ込んでくる。,My penis—now fully erect from staring at Carol's breasts—was completely engulfed by her massive chest. Despite feeling soft and plush; they wrapped around my penis with a pressure so overwhelming it caused my hips to feel like they were melting. An indescribable pleasure spread from my dick throughout my body.
```

もしもアンタがおっぱいに耐えてたら、\n結果は違ったかもしれないけど。\n頼みの綱がこれじゃ、あの子たちもみんなやられちゃう❤,If you had managed to endure my tits; things might've gone differently. But with their savior reduced to this; all those kids will all get defeated; too~❤
あーあー、パイズリなんかに負けなければよかったのに❤\nとんでもなく気持ちよくても、\nチンポこらえて射精我慢すればよかったのに❤,Oh well. Guess you shouldn't have lost to a titfuck; huh?❤ No matter how incredibly good it felt; you should have held your cock back and resisted cumming~❤
ほらほらほら❤こんな風に\nだぱんだぱんって激しくパイズリされてもさぁ、\nイクの我慢してみなよ❤,Give it a try~❤ Even getting violently titfucked like this—slap; slap; slap—just try holding your orgasm back!❤
おっぱいでチンポ溶けちゃう？\nぷぷっ、腰をふるわせてアヘアへ言いながらよだれ垂らして、\n少しは抵抗してみせろっての❤,Are my tits making your cock melt? Pfft; you're shaking your hips; moaning and drooling... Show some self-control!❤
ほら、おっぱいで脳みそどろどろになっちゃったマゾは、\nさっさとイケ❤\nサキュバス様にエナジー吸い取られろ❤きゃははは♪,Come on; you masochist; with your brain all melted by tits! Hurry up and cum.❤ Let little miss succubus suck out all your energy~❤ Kyahahaha!♪
あぐぅっ❤あっ、あああっ❤❤とまらないっ❤\nきもちいいっ❤きもちいいっ❤❤,Aghh!❤ Ah; ahh!❤❤ I-I can't stop!❤ It feels good!❤ It feels too good!!❤❤

> エナジー吸い取られて、どんどん干からびてんの気づいてる？\nしおしおになって、もうアンタ死にかけだよ❤\nサキュバスに射精してんだから当たり前だよね❤,Do you even realize your energy is getting sucked out; making you wither away more and more? You're getting all shriveled up... You're almost dead!❤ That's just what happens when you start cumming for a succubus~❤

Author: Too many words in that first and second sentence.  This is definitely a case of less is more.
```suggestion
エナジー吸い取られて、どんどん干からびてんの気づいてる？\nしおしおになって、もうアンタ死にかけだよ❤\nサキュバスに射精してんだから当たり前だよね❤,Do you even realize you're withering away from all the energy I'm sucking out of you? You're shriveling up so much... you're almost dead!❤ That's just what happens when you start cumming for a succubus~❤
```

ぷっ、挟まれたばっかりなのに、もう射精の準備始めちゃってる❤\n焦らしてたからすっごい気持ちいいでしょ？\n気持ちよすぎてチンポ溶けそうなんじゃない？,Pfft; I've just barely squeezed you between them and you're already getting ready to cum.❤ Since I've been teasing you so much; it feels extra nice; huh? Does it feel so good your cock might melt?
我慢なんてさせてあげない❤\nさっさと出してわたしを回復させてよ。ほらほらほら❤,I won't let you hold back!❤ Hurry up and cum so I can restore my energy! Come on~❤
はひぃっ・・・❤❤,Hahhh...❤❤

> はい、楽勝❤\nサキュバスのおっぱいでチンポ瞬殺されちゃったね❤\n快感強すぎて腰抜けちゃった？くすくす❤,There; easy peasy~❤ Your cock broke instantly between my succubus tits.❤ Did it feel so good your legs give out? *Giggle*❤

Author: Hmm 'broke' still sounds weird. Without proper context, it kind of sounds more like physical harm.

Reviewer: Okay I'll look into it, but it was your suggestion

Reviewer: How about something like 'You're cock gave in the instant it was between my succubus tits' or something

Author: I like that! I'll change it a little but I'll go with:
```suggestion
はい、楽勝❤\nサキュバスのおっぱいでチンポ瞬殺されちゃったね❤\n快感強すぎて腰抜けちゃった？くすくす❤,There; easy peasy~❤ Your cock caved the instant it got between my succubus tits.❤ Did it feel so good your legs give out? *Giggle*❤
```

Reviewer: 'It got' feels kinda passive. What about something like 'I stuck it between'?

Author: Yep I like that. I'll change it ^^

彼女の胸が与えたあまりに強烈な快感がまだ腰に残っている。僕の脚はぷるぷると震え、地面に尻もちをついたまま立ち上がることもできなかった。,I could still feel the intensely powerful pleasure her breasts had given me lingering in my hips. My legs trembled and shook; unable to hold my weight; forcing me to fall to the floor.
きゃはは、なにその脚の動き。どんだけだよ❤\nサキュバスになって、わたしヤバいぐらい強くなってるじゃん❤,Kyahaha; what's with those sad; shaky legs? So pathetic~❤ Now that I've become a succubus; I guess I've gotten ridiculously strong.❤
元からおっぱいおっきかったけどさー、\n今じゃ超爆乳になって、おっぱいひとつでアンタの頭より大きいよ。\nヤバくないこのデカさ❤,My tits were already big before; but now they're super massive—each ones bigger than your head! Pretty awesome; right?❤
Bランクで、無敗だった男を瞬殺❤\nアタシのマスターがずっとアンタに追いつけないのを悩んでたけど、\nぷぷっ、こんな簡単な話だったんだね❤,Big enough to kill an undefeated B-rank man; even.❤ My master was always worrying about not being able to catch back up to you; but pfft; looks like it was this easy all along~❤
サキュバスになっちゃえば、男なんてみんなザコ❤\nデカちち揺らしてちょっと遊んでやれば、\n簡単に倒せちゃう❤あーん、チョロすぎ❤,Once you become a succubus; all men turn into weaklings.❤ Just bounce your big tits and tease them a little; and deafting any one of them is a cinch~❤ Ahh; it's so pathetically easy!❤

> {主人公}さん、ダメです！諦めないで！\nそんなサキュバスの快楽になんか負けちゃダメ！,{主人公}; no! Don't give up! You can't just let pleasure from some succubus beat you!

Author: Hmm was this what I suggested?
"You can't just let pleasure from some succubus" sounds off.

Reviewer: Mostly. I changed it a little because yours felt off

Reviewer: How about something like 'you can't give into a succubus's pleasure' or something.

Author: Changed to this:

```suggestion
{主人公}さん、ダメです！諦めないで！\nそんなサキュバスの快楽になんか負けちゃダメ！,{主人公}; no! Don't give up! You can't just let a succubus beat you because they make you feel good!
```

Reviewer: See that's the version I didn't like because it just feels off to me. The receptionist is talking about how dangerous a succubus's pleasures can be, so it just feels weird to disconnect the two ideas when in the Japanese they're connected by a 'no'

It also sounds a little casual for the receptionist, so tends to be a bit better spoken

Author: "Just because it feels good" then?

```suggestion
{主人公}さん、ダメです！諦めないで！\nそんなサキュバスの快楽になんか負けちゃダメ！,{主人公}; no! You can't give up! You can't let a succubus beat you just because it feels good!
```

おっぱいでチンポ溶けちゃう？\nぷぷっ、腰をふるわせてアヘアへ言いながらよだれ垂らして、\n少しは抵抗してみせろっての❤,Are my tits making your cock melt? Pfft; you're shaking your hips; moaning and drooling... Show some self-control!❤
ほら、おっぱいで脳みそどろどろになっちゃったマゾは、\nさっさとイケ❤\nサキュバス様にエナジー吸い取られろ❤きゃははは♪,Come on; you masochist; with your brain all melted by tits! Hurry up and cum.❤ Let little miss succubus suck out all your energy~❤ Kyahahaha!♪
あぐぅっ❤あっ、あああっ❤❤とまらないっ❤\nきもちいいっ❤きもちいいっ❤❤,Aghh!❤ Ah; ahh!❤❤ I-I can't stop!❤ It feels good!❤ It feels too good!!❤❤

> エナジー吸い取られて、どんどん干からびてんの気づいてる？\nしおしおになって、もうアンタ死にかけだよ❤\nサキュバスに射精してんだから当たり前だよね❤,Do you even realize you're withering away from all the energy I'm sucking out of you? You're shriveling up so much... you're almost dead!❤ That's just what happens when you start cumming for a succubus~❤

Author: "you're almost dead!" feels like a weird phrase to interject here... Hmm...

Reviewer: Then how about something like 'you're basically dead already,' or something.

Author: Hmm maybe 'nearly dead' would work better.

```suggestion
エナジー吸い取られて、どんどん干からびてんの気づいてる？\nしおしおになって、もうアンタ死にかけだよ❤\nサキュバスに射精してんだから当たり前だよね❤,Do you even realize you're withering away from all the energy I'm sucking out of you? You're shriveling up so much... you're nearly dead already!❤ That's just what happens when you start cumming for a succubus~❤
```
